src/Search.js

```diff
@@ -0,0 +1,20 @@
+import React from "react";
+import { key } from "../src/secrets";
+
+export default function Search() {
+  const api = `https://api.themoviedb.org/3/search/multi?api_key=${key}&language=en-US&&page=1&include_adult=true`;
+
+  const searchMulti = (setMulti) => {
+    fetch(api)
+      .then((response) => response.json())
+      .then((data) => setMulti(data));
```

Comment:
Could you come up with a better name for setMulti and searchMulti? This is an array of movie data from the api? Maybe setFilms?

Reply:
changed to searchMovies for the fetch request and to setSearchInput for the state and handleChange